site/content/contribute/server/developer-setup/arch.md

```diff
@@ -7,14 +7,20 @@
      newgrp docker
      ```
 
-2. Add `dockerhost` as an alias for `127.0.0.1` for use by various Mattermost build scripts:
-
-   * Edit `/etc/hosts` file as an administrator (e.g. `sudo`) and add the following line:
-
-        ```sh
-        127.0.0.1    dockerhost
-        ```
+    **Note:** [MM-9791](https://github.com/mattermost/mattermost-server/pull/10872) introduced using [docker-compose](https://docs.docker.com/compose/)  to manage containers.  When upgrading from an existing installation, to remove the old containers use `make clean-old-docker`.
+    
+    Alternatively, migration of database container data requires some manual steps
+    ```
+    #mysql
+    mysqldump -h 127.0.0.1 --column-statistics=0 -u mmuser -p mattermost_test > mm_mysql_backup.sql
```

Comment:
I know I suggested simplifying this to avoid the extra `make` invocations, but I wonder if we should call out the need to execute these steps separately. Something like:

----------

First, backup from any existing containers:
```
    mysqldump -h 127.0.0.1 --column-statistics=0 -u mmuser -p mattermost_test > mm_mysql_backup.sql
    pg_dump -U mmuser -W -d mattermost_test -h 127.0.0.1 > mm_postgres_backup.bak
```

Then after upgrading and starting the new docker-compose-managed containers, restore the data:
```
    mysql -u mmuser -p -h 127.0.0.1 mattermost_test < mm_mysql_backup.sql
    psql -U mmuser -W -h 127.0.0.1 -f mm_postgres_backup.bak mattermost_test
```

Reply:
I like it.  Happy to do it this way.  Doesthis sound OK @crspeller?

Reply:
Yes that makes sense.